project/overview.md

```diff
@@ -0,0 +1,37 @@
+<!-- vim: ft=markdown colorcolumn=72
+-->
+# Project Overview
+
+The spire/helm-chart effort intends to deliver a set of helm charts to
+support SPIRE deployments in Kubernetes to widest possible audience of
+users.
+
+## Mission and Scope
+This mission of this effort is to provide a stable upstream repository
+of Helm Charts sufficient for any organization or individual to use.
+
+### What problem does this project address?
+Prior to this effort, no single authoritative repository for SPIRE Helm
+Charts existed, imposing an additional burden on SPIRE adoption as each
+organization or individual had to develop their own Kubernetes
+deployment packaging.
+
+### What is the goal of this project?
+The goal of this project is to be the primary means of deployment for
```

Comment:
```suggestion
The goal of this project is to be a means of deployment for
```

Reply:
We have the branding, we have the lineage, and goals are aspirational, so let's shoot for the stars.  If we miss, at least our arrow will have travelled further.